src/components/Table.tsx

```diff
@@ -395,11 +407,15 @@ export default class Table<T> extends React.Component<
 					{this.props.tbodyPrefix}
 					{map(this.sortData(data), (row, i) => {
 						const isChecked = this.props.onCheck ? this.isChecked(row) : false;
+						const isHighlighted = this.props.highlightedRow
+							? this.isHighlighted(row)
+							: false;
```

Comment:
It'd be a little nicer to do this check inside isHighlighted. Right now, you have to know that calling it without checking this first might do unpredictable things.

Reply:
good idea